src/scripts/main.js

```diff
@@ -1,3 +1,40 @@
 'use strict';
 
-// write code here
+const headers = document.querySelectorAll('th');
+const tBody = document.querySelector('tBody');
+const rows = document.querySelectorAll('tBody tr');
+
+const arrayOf = [...rows];
+
+headers.forEach((header) => {
+  header.addEventListener('click', function () {
+    const headerIndex = event.target.cellIndex;
+
+    sortBy(headerIndex);
+  });
+});
+
+function sortBy(headerIndex) {
+  arrayOf.sort((td1, td2) => {
+    if (
+      headers[headerIndex].innerHTML === 'Name' ||
+      headers[headerIndex].innerHTML === 'Position'
+    ) {
```

Comment:
It's not a good practice to hardcode specific header names in your function. What if the headers change? Your function will break. Try to make your function more generic.